### PR TITLE
TTK-18320 add command to list pics by some filters

### PR DESCRIPTION
**Pull request by Yurujai**

### CODE REVIEW

File: src/Pumukit/EncoderBundle/Command/PumukitPicsConvertCommand.php [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    private $convert_quality;
+    private $no_replace;
+    private $convert_max_width;
+    private $convert_max_height;
+
+    protected function configure()
+    {
+        $this
+            ->setName('pumukit:pics:convert')
+            ->setDescription('Pumukit list pics')
```
Comment (mreyk) [inline src/Pumukit/EncoderBundle/Command/PumukitPicsConvertCommand.php]: @Yurujai Change description of command.

File: src/Pumukit/EncoderBundle/Command/PumukitListPicsCommand.php [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        $validInput = $this->checkInputOptions();
+        if (!$validInput['success']) {
+            throw new \Exception($validInput['message']);
+        }
+
+        try {
+            $inputs = $this->picService->formatInputs($this->id, $this->size, $this->path, $this->extension, $this->tags, $this->exists, $this->type);
+            list($this->id, $this->size, $this->path, $this->extension, $this->tags, $this->exists, $this->type) = $inputs;
+        } catch (\Exception $exception) {
+            throw new \Exception($exception->getMessage());
```
Comment (rubenrua) [inline src/Pumukit/EncoderBundle/Command/PumukitListPicsCommand.php]: Why throw a new exception.

File: src/Pumukit/EncoderBundle/Command/PumukitListPicsCommand.php [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+    /**
+     * @param $data
+     *
+     * @return bool
+     */
+    private function showData($data)
+    {
+        if (empty($data['pics'])) {
+            $this->output->writeln('No pics found');
```
Comment (rubenrua) [inline src/Pumukit/EncoderBundle/Command/PumukitListPicsCommand.php]: 2nd time same msg. See line 111

File: src/Pumukit/EncoderBundle/Command/PumukitListPicsCommand.php (hunk 2 above)
Comment (mreyk) [reply src/Pumukit/EncoderBundle/Command/PumukitListPicsCommand.php]: @Yurujai ?

File: src/Pumukit/EncoderBundle/Command/PumukitListPicsCommand.php (hunk 3 above)
Comment (mreyk) [reply src/Pumukit/EncoderBundle/Command/PumukitListPicsCommand.php]: @Yurujai ?

**Pull request merged**